docs/_blog/announcing-anchor-outputs-channels.md

```diff
@@ -0,0 +1,66 @@
+Before the anchor outputs feature, participants continually negotiated their commitment
+transaction’s fees based on the current blockspace demand to ensure a timely confirmation if they
+were to be broadcast. This fee negotiation unfortunately came with its own set of problems. If
+participants disagreed on the fees proposed for a channel, the channel became unusable due to a
+unilateral close. Underestimating could result in loss of funds if an HTLC needs to be resolved
+onchain, and overestimating would allocate the majority of the channel balance towards fees,
+potentially rendering the channel useless.
+
+With anchor outputs, participants are now able to contribute the majority of fees required at the
+time of broadcast by bumping the commitment transaction’s fee via a child transaction using the
+Child-Pays-For-Parent (CPFP) fee-bumping mechanism. A small portion of fees must still be allocated
+to commitment transactions to ensure they can enter nodes’ mempools on their own. This will be
+required until package relay is deployed network-wide, at which point we can have a fixed 1 sat/vB
+commitment transaction and do away with the fee negotiation once and for all, eliminating the most
+common cause behind unilateral closes we see today.
+
+Given LDK’s inherent design, supporting anchor outputs posed a few challenges. As a library, we
+cannot expect to easily have access to an onchain wallet as a traditional node implementation would
+without some manual integration work by the application developer. We were able to leverage our
+existing [event-based API](https://docs.rs/lightning/latest/lightning/events/index.html) to
+communicate to users that a commitment or HTLC transaction's fee needs to be bumped. The handling of
+such events can be non-trivial, so we've added a utility
+([`BumpTransactionEventHandler`](https://docs.rs/lightning/latest/lightning/events/bump_transaction/struct.BumpTransactionEventHandler.html))
+to simplify the integration work required. With this utility, an application developer only needs to
+provide a view into their onchain wallet for LDK to access confirmed UTXOs and sign them.
+
+Note that node operators now need to maintain a reserve of confirmed funds readily available to
+spend if they wish to exit a channel unilaterally. Currently, LDK will not ensure this reserve is
+enforced, but we’re exploring possible ways to do so at
+https://github.com/lightningdevkit/rust-lightning/issues/2320. Without such enforcement, a node may
+open/accept more channels than its reserve is meant to handle, potentially resulting in a loss of
+funds if any HTLCs need to be resolved onchain. In the meantime, application developers will need to
+determine whether their use case warrants such enforcement and implement it themselves. For example,
+a mobile user connected to a LSP could always defer to the LSP to broadcast the latest state such
```

Comment:
I think a better example can be to say “a mobile user could always defer the fee-bumping of its commitment or HTLC transactions to a third-party watchtowers”.

Ideally you wouldn’t let the LSP with whom you have an opened channel be responsible of confirming your time-sensitive state, as they have a competing interest not to do so. However a crowd of watchtower sounds more adequate.